spec/http_spec.rb

```diff
@@ -10,7 +10,7 @@
   let(:retriable_intervals) { [] }
   let(:public_routes) { false }
 
-  let(:response) { instance_double('Faraday::Response', body: body) }
+  let(:response) { instance_double('Faraday::Response', body:) }
```

Comment:
I guess this is something different you fixed in this PR
It's fine just check it's not by error.

Reply:
Yeah, rubocop fixed it :)